src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```diff
@@ -112,6 +112,7 @@
     <_AggressiveAttributeTrimming Condition="'$(_AggressiveAttributeTrimming)' == ''">true</_AggressiveAttributeTrimming>
     <NullabilityInfoContextSupport Condition="'$(NullabilityInfoContextSupport)' == ''">false</NullabilityInfoContextSupport>
     <BuiltInComInteropSupport Condition="'$(BuiltInComInteropSupport)' == ''">false</BuiltInComInteropSupport>
+    <JsonSerializerIsReflectionEnabledByDefault Condition="'$(JsonSerializerIsReflectionEnabledByDefault)' == ''">true</JsonSerializerIsReflectionEnabledByDefault>
```

Comment:
shouldthis not check '$(TrimMode)' as well? given the commit message this should be set only when '$(TrimMode)' is 'partial'.

Reply:
cc @eerhardt

Reply:
I think this makes sense too. If someone sets `TrimMode=full`, JsonSerializer is basically guaranteed to break. So you might as well disable reflection and guide them to use the source generator.

Reply:
IIRC we didn't add a similar condition in Blazor. Do we need to revisit there?

Reply:
Blazor doesn't support `TrimMode=full` at all. So I don't think we need to worry about it for .NET 8.